### ISSUE TITLE
Retry `noarch: python`

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/pyelftools-feedstock#32]
It looks like `noarch: python` has been tried here before, but was disabled as there were scripts in the `recipe` directory

https://github.com/conda-forge/pyelftools-feedstock/blob/14550f74647a6eeae0862df6cc8ad10942b052a9/recipe/meta.yaml#L13-L17

However it looks like there are no scripts in [the `recipe` directory currently]( https://github.com/conda-forge/pyelftools-feedstock/tree/14550f74647a6eeae0862df6cc8ad10942b052a9/recipe )

So could we try making this `noarch: python` again? This would allow the same packages to be installed on other platforms

**Comment by jakirkham** [comment]
Or am I misunderstanding that comment and it is referring to [the `scripts` in `setup.py`]( https://github.com/eliben/pyelftools/blob/0f72cb7990c7039258b7988602b0aec83c5864f7/setup.py#L47 )?

Here is the relevant line from `setup.py`

```python
    scripts=['scripts/readelf.py'],
```

**Comment by ocefpaf** [comment]
> Or am I misunderstanding that comment and it is referring to [the `scripts` in `setup.py`](https://github.com/eliben/pyelftools/blob/0f72cb7990c7039258b7988602b0aec83c5864f7/setup.py#L47)?

Yep. Those are problematic to conda. I'm not sure if this is still an issue. If my memory is correct the problem is that the script is not added to the path on Windows and one must call it using the full path.
